fcrepo-http-api/src/test/java/org/fcrepo/http/api/FedoraTombstonesTest.java

```diff
@@ -69,6 +69,6 @@ public void testDelete() {
         final Response actual = testObj.delete();
         assertEquals(NO_CONTENT.getStatusCode(), actual.getStatus());
         verify(mockResource).delete();
-        verify(mockSession).commit();
+        verify(mockTransaction).commit();
```

Comment:
tests will need to account for shortlived transactions too

Reply:
@mohideen :  Please add a JIRA to track this detail.

Reply:
https://jira.duraspace.org/browse/FCREPO-3093